### DISCUSSION TITLE
Where is hosted now cloud-pricing-api

### DISCUSSION THREAD

**Discussion by camaeel** [infracost/infracost#2945] [Q&A]
Docker hub registry: https://hub.docker.com/r/infracost/cloud-pricing-api points to non-existent (or not public) github repo: https://github.com/infracost/cloud-pricing-api.
What is the current faith of this application? Was it moved to another repo or it is no longer available publicly? 

BTW: Is there any chance for arm64 support for this tool?

**Comment by hassankhosseini** [comment]
Hey Camaeel, the self-hosting option moved to the paid plan last year. You can still use the hosted version for free by pointing to the public pricing API. Docs here: https://www.infracost.io/docs/supported_resources/cloud_pricing_api/
Alternatively, please email hello@infracost.io and we can discuss pricing for the enterprise self hosted version. Thanks!
